CHANGELOG.md

```diff
@@ -8,18 +8,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## [0.4.0]
 
-### Changed
+### Added
 
-- Changed default labels for Liberty Logging to disable tracing to container
-  logs ([#95](https://github.com/OpenLiberty/open-liberty-operator/issues/95))
+- Added support for single sign-on using social login providers and any OIDC & OAuth 2.0 based clients. ([#123](https://github.com/OpenLiberty/open-liberty-operator/pull/122))
+- Added support for integration with OpenShift's Certificate Management. ([#122](https://github.com/OpenLiberty/open-liberty-operator/pull/122))
```

Comment:
We do integrate with OpenShift CA, we integrate with cert-manager (Certificate CRD).

Reply:
Addressed. Thanks.